CONTRIBUTING.md

```diff
@@ -47,9 +44,17 @@ are good for new contributors.
 These guidelines seek to prioritize efforts that would benefit the community the most.
 Feedback is welcome.
 
+### Style and Practices
+* Please refer to [TensorFlow's style guide](https://www.tensorflow.org/community/contribute/code_style). Don't forget to run pylint.
+
+Unless agreed upon with the project maintainer in the issue or PR, the following are necessary to
+merge a PR.
+* Unit tests for behavioral changes. Keep in mind that unit tests that take a long time to run also make it harder to contribute.
```

Comment:
What is counted as 'a long time'?

Reply:
That's a good question. I've done it in the past, but it would be worth running the unit tests on a standard developer laptop and see how long the longest ones take.

I hope that at the least, this will encourage people to think a bit more before trying to have a unit test that takes >5 - 6 minutes by training on a larger model for multiple epochs, when a smaller model with one epoch would achieve the same test coverage.